src/TrackGenerator.cpp

```diff
@@ -1825,66 +1929,71 @@ void TrackGenerator::splitSegments(FP_PRECISION max_optical_length) {
     log_printf(ERROR, "Unable to split segments since "
 	       "tracks have not yet been generated");
 
-  int num_cuts, min_num_cuts;
-  segment* curr_segment;
+#pragma omp parallel
```

Comment:
Ditto with this new code - I'm not sure that any of our tests actually require segment splitting. I think we need a test problem that meets the segment splitting criterion in order to verify this.